14-draft.json

```diff
@@ -1132,6 +1132,49 @@
           "end"
         ]
       }
+    },
+    "membership_plans": {
+        "description": "A list of the different membership plans your hackerspace might have.",
+        "type": "array",
+        "items": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "The name of the membership plan. For example Student Membership, Normal Membership etc...",
```

Comment:
Minor nitpick: I would use either etc. or ...:
* "The name of the membership plan. For example Student Membership, Normal Membership, ..."
* "The name of the membership plan. For example Student Membership, Normal Membership, etc."